### PR TITLE
Refs 2237 : Create UI home screen

### PR DESCRIPTION
**Pull request by yenntt-1728**
https://edu-redmine.framgia.vn/issues/2237

### CODE REVIEW

**Comment by yenntt-1728** [comment]
ready

**Review by NguyenMinhTuanB — APPROVED** [review]

**Comment by NguyenMinhTuanB** [comment]
merged

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -1,18 +1,49 @@
...
+            type="com.yennguyen.yen.moviedb_27.screen.main.MainViewModel" />
+    </data>
+
+    <android.support.constraint.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent">
+
+        <android.support.v7.widget.Toolbar
+            android:id="@+id/toolbar"
+            android:layout_width="match_parent"
```
Comment (hoangth1) [inline app/src/main/res/layout/activity_main.xml:20]: Constrain layout không enen đặt match_parent nhé

File: app/src/main/res/layout/activity_main.xml [hunk 2]
```diff
@@ -1,18 +1,49 @@
...
+            android:layout_height="wrap_content"
+            android:background="@color/colorApplication"
+            app:theme="@style/ThemeOverlay.AppCompat.Dark"
+            app:title="@string/app_name" />
+
+        <FrameLayout
+            android:id="@+id/frame_container"
+            android:layout_width="0dp"
+            android:layout_height="match_parent"
```
Comment (hoangth1) [inline app/src/main/res/layout/activity_main.xml:29]: đây nữa

**Pull request merged**